ibm/service/db2/data_source_ibm_db2_instance.go

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package db2
+
+import (
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/resourcecontroller"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceIBMDb2Instance() *schema.Resource {
+	riSchema := resourcecontroller.DataSourceIBMResourceInstance().Schema
+
+	riSchema["high_availability"] = &schema.Schema{
+		Description: "If you require high availability, please choose this option",
+		Optional:    true,
+		Type:        schema.TypeString,
+	}
+
+	riSchema["machine_type"] = &schema.Schema{
+		Description: "Avaialble machine type flavours (default selection will assume dev config of 1 core & 4GB of memory for cost saving)",
```

Comment:
Change to "Available machine type flavours (default selection will assume smallest configuration)"